index.js

```diff
@@ -1,8 +1,9 @@
 'use strict'
 
-var url = require('url')
 var isUrl = require('is-url')
 
+var laxUrlRegex = /(?:(?:[a-z]+:)?[/][/])?([^/]+)([/][^?#]+)/
```

Comment:
Regex description:

- match protocol if it exists
- match everything until `/` as `hostname`
- match everything until `?`, `#` or EOF as `pathname`

We're only matching `github.com` URLs so the URLs that matter won't be esoteric.